src/webui/api/torrentscontroller.cpp

```diff
@@ -738,7 +738,7 @@ void TorrentsController::setLocationAction()
     const QString newLocation {params()["location"].trimmed()};
 
     if (newLocation.isEmpty())
-        throw APIError(APIErrorType::BadParams, tr("Save path is empty"));
+        throw APIError(APIErrorType::BadParams, tr("Save path cannot be empty"));
```

Comment:
I see it isn't your code, but why "save path" can't be empty? AFAIK, empty string here just means "use application default save path".

Reply:
Well, forget it. Anyway, it's a separate topic.